debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/RecordMaker.java

```diff
@@ -16,26 +19,27 @@
     private final boolean emitTombstoneOnDelete;
     private final Filters filters;
     private final SourceInfo sourceInfo;
+    private static final Logger LOGGER = LoggerFactory.getLogger(RecordMaker.class);
```

Comment:
Could you put the `static` field before all the instance fields?

Reply:
Updated accordingly~